src/PageBanner/PageBanner.scss

```diff
@@ -0,0 +1,47 @@
+.pgn__pageBanner-component {
+  width: 100%;
+  min-height: 36px;
+  max-height: 56px;
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.pgn__pageBanner__dark {
+  background-color: $dark-500;
+  color: white;
+}
+
+.pgn__pageBanner__light {
+  background-color: $light-400;
+  color: black;
+}
+
+.pgn__pageBanner__accentA {
+  background-color: $accent-a;
+  color: black;
+}
+
+.pgn__pageBanner__warning {
+  background-color: $warning-100;
+  color: black;
+}
+
+.pgn__pageBanner-content {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  padding: 8px 12px;
```

Comment:
nit: Should the padding values come from the `$spacers` map? Example:

```
padding: map.get($spacers, 2) map.get($spacers, 2.5);
```

https://github.com/edx/paragon/blob/a861078c6e4dc452ab580dbdcfdbe46c632fa6c6/scss/core/_variables.scss#L360-L376

Reply:
Yes I didn't think to use these! Thanks